client/extensions/woocommerce/woocommerce-services/views/label-settings/add-credit-card-modal.js

```diff
@@ -21,48 +20,45 @@ import { addStoredCard } from 'state/stored-cards/actions';
 import { createCardToken } from 'lib/store-transactions';
 import analytics from 'lib/analytics';
 
-class AddCardDialog extends Component {
-	static propTypes = {
-		siteId: PropTypes.number.isRequired,
-		isVisible: PropTypes.bool,
-		addStoredCard: PropTypes.func.isRequired,
-		closeAddCardDialog: PropTypes.func.isRequired,
-	};
-
-	constructor( props ) {
-		super( props );
-		this.createCardToken = curry( createCardToken )( 'card_add' );
-	}
-
-	recordFormSubmitEvent() {
+function AddCardDialog( {
+	siteId,
+	isVisible,
+	translate,
+	closeAddCardDialog: closeDialog,
+	addStoredCard: saveStoredCard,
+} ) {
+	const createCardAddToken = ( ...args ) => createCardToken( 'card_add', ...args );
+	const recordFormSubmitEvent = () =>
 		analytics.tracks.recordEvent( 'calypso_add_credit_card_form_submit' );
-	}
-
-	render() {
-		const { siteId, isVisible, translate } = this.props;
+	const onClose = () => closeDialog( siteId );
 
-		const onClose = () => this.props.closeAddCardDialog( siteId );
-
-		return (
-			<Dialog
-				additionalClassNames="add-credit-card-modal woocommerce wcc-root"
-				isVisible={ isVisible }
-				onClose={ onClose }
-			>
-				<CreditCardForm
-					createCardToken={ this.createCardToken }
-					recordFormSubmitEvent={ this.recordFormSubmitEvent }
-					saveStoredCard={ this.props.addStoredCard }
-					successCallback={ onClose }
-					showUsedForExistingPurchasesInfo={ true }
-					heading={ translate( 'Add credit card' ) }
-					onCancel={ onClose }
-				/>
-			</Dialog>
-		);
-	}
+	return (
+		<Dialog
+			additionalClassNames="add-credit-card-modal woocommerce wcc-root"
+			isVisible={ isVisible }
+			onClose={ onClose }
+		>
+			<CreditCardForm
+				createCardToken={ createCardAddToken }
+				recordFormSubmitEvent={ recordFormSubmitEvent }
+				saveStoredCard={ saveStoredCard }
+				successCallback={ onClose }
+				showUsedForExistingPurchasesInfo={ true }
+				heading={ translate( 'Add credit card' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Add Credit Card' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woo-add-card-modal-functional)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Safe to ignore.